.github/workflows/deploy.yml

```diff
@@ -38,6 +38,15 @@ jobs:
           cf_username: ${{secrets.CF_USER}}
           cf_password: ${{secrets.CF_PASSWORD}}
           command: add-network-policy funding-service-design-assessment-store-dev funding-service-design-notification-dev --protocol tcp --port 8080
+    - name: Apply network policy for application store service for dev
+      uses: citizen-of-planet-earth/cf-cli-action@v2
+      with:
+          cf_api:      ${{secrets.CF_API}}
+          cf_org:      ${{secrets.CF_ORG}}
+          cf_space:    ${{secrets.CF_SPACE }}
+          cf_username: ${{secrets.CF_USER}}
+          cf_password: ${{secrets.CF_PASSWORD}}
+          command: add-network-policy funding-service-design-assessment-store-dev funding-service-design-application-store-dev --protocol tcp --port 8080
```

Comment:
Are we using this network policy? Doesn't seem like any reference to application-store in the code?

Reply:
In preperation for when we query the app store for the jsons.

Reply:
But will move it out for now and include in the PR where that functionality is added, if that is agreeable.